dss-api.yml

```diff
@@ -1023,32 +1020,79 @@ definitions:
     required:
       - uuid
       - versions
+  NotificationEndpoint:
+    type: object
+    properties:
+      url:
+        description: The URL which an HTTP request is sent to when DSS delivers a notification to this endpoint.
+        type: string
+        pattern: 'https?://'
+      method:
+        description: The HTTP request method to use when DSS delivers a notification to this endpoint.
+        enum: [POST, PUT]
+      encoding:
+        enum:
+          - application/json
+          - multipart/form-data
+          - application/x-www-form-urlencoded
+        description: >
+          The MIME type describing the encoding of the request body
+          * `application/json` - the HTTP request body is the notification payload as JSON
+          * `multipart/form-data` - the HTTP request body is a list of form fields, each consisting of a name and a
+            corresponding value. See https://tools.ietf.org/html/rfc7578 for details on this encoding. The actual
+            notification payload will be placed as JSON into a field of the name specified via `payload_form_field`.
+          * `application/x-www-form-urlencoded` - he HTTP request body is a list of form fields, each consisting of
+            a name and a corresponding value. See https://www.w3.org/TR/html401/interact/forms.html#h-17.13.4.1
+      form_fields:
+        description: |
+          A collection of form fields to be supplied in the request body. Since the type of this field is `object`,
+          multi-valued fields are not supported.  Ignored unless `encoding` is `multipart/form-data` or
+          `application/x-www-form-urlencoded`.
```

Comment:
Agreed.

Reply:
Added that qualifier and an example. The example looks a bit odd because form-encoding is not exactly human readable, but hey, at least it's not a hexdump of binary data.